### ISSUE TITLE
lstat doesn't exist on windows

### ISSUE THREAD

**Issue by awatson1978** [awatson1978/starrynight#59]
Re: robfallows

TL;DR: I got StarryNight installed, but couldn't get it to run. YMMV.

So, I finally have something to report wrt Windows. I stripped Windows back to a bare installation to find out what a Windows developer might need to install prior to installing StarryNight.

**Prerequisites:**  
1.  **nodejs** and **npm**. Don't do what I did and install the latest version of node. Things will break. I eventually installed our old friend version 0.10.36. I used the msi file from http://nodejs.org/dist/v0.10.36/, which also installs npm 1.4.28.
2.  **python**. I installed version 2.7 from https://www.python.org/downloads/release/python-2710/. Version 3.4.3 might work - I didn't try.
3.  You need a **C** compiler. I installed Microsoft Visual Studio Community 2013 from https://www.visualstudio.com/products/visual-studio-community-vs. Other C compilers might work, I don't know. The errors I saw prior to installing this referenced Visual Studio, so I took the route of least pain (but probably longest ever installation time). You should check the licence if you are using it commercially.
4.  Check your **PATH** settings. node and npm seemed to set this up for me. So did Visual Studio, after a reboot, but python didn't. If you don't know where to check/set, click the Windows button, right click on Computer, select properties -> advanced system settings -> Environment Variables (Win7).
   From here, you can now open up a command prompt and install starrynight:

`npm install -g starrynight`

During this I got a handful of warnings:

``` bash
warning C4506: no definition for inline function 'v8::Persistent<v8::Object> v8::Persistent<v8::Object>::New(v8::Handle<v8::Object>)' [C:\users\***\AppData\Roaming\npm\node_modules\starrynight\node_modules\pioneer\node_modules\selenium-webdriver\node_modules\ws\node_modules\bufferutil\build\bufferutil.vcxproj]
```

``` bash
warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc [C:\users\***\AppData\Roaming\npm\node_modules\starrynight\node_modules\mrtbulkrelease\node_modules\sync-prompt\build\sync_prompt.vcxproj]
```

However, the installation completed ok smile, although on my core i5 laptop it took 3 hours worried!

Unfortunately, when I tried to add the nightwatch scaffolding:

`starrynight scaffold --framework nightwatch`

I got the following error:

Starting scaffolding of nightwatch tests...

``` bash
{ [Error: ENOENT, lstat 'C:\Users\rfallows\AppData\Roaming\npm\lib\node_modules\starrynight\scaffolds\sample-tests\nightwatch']
  errno: 34,
  code: 'ENOENT',
  path: 'C:\\Users\\rfallows\\AppData\\Roaming\\npm\\lib\\node_modules\\starrynight\\scaffolds\\sample-tests\\nightwatch' }
```

The issue seems to be with no lstat command in Windows. I have checked cygwin, GnuWin and MinGW and cannot find a Windows equivalent. Currently stuck!
